src/Grpc.AspNetCore.Server/GrpcServiceExtensions.cs

```diff
@@ -62,6 +62,7 @@ public static IGrpcServerBuilder AddGrpc(this IServiceCollection services)
 
             services.AddRouting();
             services.AddOptions();
+            services.AddLogging();
```

Comment:
For easy testing? Nothing else in our stack does this.

Reply:
I noticed AddGrpc by itself errors. Is that fine?

Reply:
Logging is added by the host, why does it error?

Reply:
It errored by itself in a test. If every other framework leaves it to the host then I'll remove it.